src/react/workflow/details/Configuration.jsx

```diff
@@ -30,13 +31,14 @@ function Configuration({ wfSelected, replications, bucketList, locations, showEd
         return replications.find(r => r.streamId === workflowId);
     }, [replications, workflowId]);
 
-    const deleteWorkflow = (item) => {
-        dispatch(deleteReplication(item));
+    const handleOpenDeleteWorkflowModal = () => {
+        dispatch(openWorkflowDeleteModal());
     };
 
     // TODO: Adapt it to handle the other workflow types; For now only replication workflow is supported.
     return (
         <TableContainer>
+            <WorkflowDeleteModal workflow={wfSelected}/>
```

Comment:
Let's use `ui-elements/DeleteConfirmation.jsx` component.

Reply:
If I use `ui-elements/DeleteConfirmation.jsx` it will not match exactly with the Modal provided by Valamir. Is it okay ?

Modal with DeleteConfirmation:
![image](https://user-images.githubusercontent.com/44285344/119496656-ff71eb80-bd63-11eb-9af0-1b7ae633be94.png)

Reply:
I will push for having a consistent confirmation modal. @Cuervino Is it ok if we stay consistent?

Reply:
Yes, indeed here the consistency makes a lot of sense. 👍